File: self_checkout/python/medeirosthiago/self_checkout.py
def main():
    subtotal = 0
    item = 0
    for item in range(3):
        item += 1
        price = input('Enter the price of item {}: '.format(item))
        qnty = input('Enter the quantity of item {}: '.format(item))
        subtotal += int(price) * int(qnty)

    tax = subtotal * 0.055
    total = subtotal + tax

    print('Subtotal: ${0:.2f}'.format(subtotal))
    print('Tax: ${0:.2f}'.format(tax))
    print('Total: ${0:.2f}'.format(total))


if __name__ == '__main__':
    main()

File: self_checkout/ruby/frigginglorious/self_checkout.rb

items = []
quantity = []
subTotal = 0
total = 0

# print "How many items are "
# subTotalItems = gets.chomp.to_i

input = ""
i = 0

until input == "done" do # I guess this is superflous
  i += 1
  printf "%-40s #%s: $", "Enter item price, or 'done' to quit", i
  input = gets.chomp.downcase
  break if input == "done"
  items.push(input.to_f)
  printf "%-40s #%s: ", "Enter the quantity of item" , i
  quantity.push(gets.chomp.to_i)
end



i = 0
until i >= items.count do
  subTotal += (items[i] * quantity[i])
  i += 1
end

tax = subTotal * 0.055

total = subTotal + tax

printf("%-20s $%0.2f \n", "Subtotal: ", subTotal)
printf("%-20s $%0.2f \n", "Tax: " ,tax)
printf("%-20s $%0.2f \n", "Total: ", total)
